### ISSUE TITLE
List all zones in the condition section 

### ISSUE THREAD

**Issue by bbiegun** [nielsfaber/scheduler-card#495]
### Checklist

- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.
- [X] I will participate in further discussion about this feature and can test it (if requested) once it's done.

### Proposal

Proposal: 
to list all of the defined zones (including passive ones) in the list in the condition based on a ``person`` entity.

Current status:
When building a condition based on a ```person``` entity we can only choose if they're in (or not) in home or outside of home. It's mapped to the ```home``` and ``not_home`` values of the entity. Outside of home means that person isn't in any of the defined zones. 

In a multizone envirionement number of possible conditions based on the person location is very limitted. 
We can check if the person 

- is or isn't in ```home``` zone
- is or isn't in any of defined zones.

### Additional info

Preview:
![Zrzut ekranu z 2021-12-28 12-38-21](https://user-images.githubusercontent.com/23291281/147562669-86213ba0-98fb-40ed-9190-78e06b2e2a82.png)

Current workaround:
I find a way to workaround it by customising the scheduler. It's working, but it requires updating configuration every time you're adding/changing zones.
```
type: custom:scheduler-card
customize:
  person.*:
    states:
      - home
      - not_home
      - zone1
      - zone2
```

**Comment by nielsfaber** [comment]
I am not using the person integration myself, the `home` and `not_home` states are what I could find in the code.
Do the person entities advertise all options for the additional zones in some property?
Otherwise, where can scheduler-card find the list of possible zones?
Is this related to the [zone entity type](https://www.home-assistant.io/integrations/zone/) as well?
If you could point me to some documentation regarding the relations of the persons and zones, that would be helpful.

**Comment by bbiegun** [comment]
Zone entity is basically what I'm looking after. `person` would advertise only current zone they're in.
I found another [lovelace plugin code](https://github.com/thomasloven/lovelace-auto-entities/blob/9961ee0c56b22b46af0234857a019e47954b6c0f/src/filter.ts#L58) which is capable of listing all zones. We're looking to list all entities where `domain == zone`. 
I hope this TS make some sense for you.

Example from lovelace-auto-entities plugin:
![image](https://user-images.githubusercontent.com/23291281/151871366-c0d76d7e-1a8d-4d47-8fa1-1a16457e2bfa.png)

**Comment by nielsfaber** [comment]
I released v2.3.3 which should add the zones to the `person` entities.
Please test if this is the case, if so you can close the issue.
Otherwise please let me know.

**Comment by bbiegun** [comment]
Works like a charm. Thank you a lot!
